addons/im_livechat/views/discuss_channel_views.xml

```diff
@@ -45,8 +45,8 @@
             <field name="arch" type="xml">
                 <list js_class="im_livechat.discuss_channel_list" sample="1" string="History" create="false" default_order="create_date desc">
                     <field name="create_date" string="Date"/>
-                    <field name="livechat_agent_history_ids" string="Agents" widget="many2many_tags_avatar" optional="show"/>
-                    <field name="livechat_bot_history_ids" string="Bot" widget="many2many_tags_avatar" optional="show"/>
+                    <field name="livechat_agent_partner_ids" string="Agents" widget="many2many_tags_avatar" optional="show"/>
+                    <field name="livechat_bot_partner_ids" string="Bot" widget="many2many_tags_avatar" optional="show"/>
```

Comment:
The problem with using these fields rather than history is that they are not working if the corresponding records are deleted.